lib/modules/newCommentCount.js

```diff
@@ -268,18 +269,20 @@ async function processCommentCounts(ele = document.body) {
 	}
 	const IDre = /\/r\/[\w]+\/comments\/([\w]+)\//i;
 	const commentsLinks = ele.querySelectorAll('.sitetable.linklisting div.thing.link a.comments');
-	Array.from(commentsLinks).forEach(link => {
-		const href = link.getAttribute('href');
+	commentsLinks::forEachChunked(link => {
+		const $link = $(link);
+		const href = $link.attr('href');
 		// split number from the word comments
-		const thisCount = link.textContent.replace(/\D/g, '');
+		const thisCount = $link.text().replace(/\D/g, '');
 		const matches = IDre.exec(href);
 		if (matches) {
 			const thisID = matches[1];
 			if ((typeof commentCounts[thisID] !== 'undefined') && (commentCounts[thisID] !== null)) {
 				const diff = thisCount - commentCounts[thisID].count;
 				if (diff > 0) {
-					const $newString = $(`<span class="newComments">&nbsp;(${diff} new)</span>`);
-					$(link).append($newString);
+					$link.append(`<span class="newComments">&nbsp;(${diff} new)</span>`);
+					// add .res-hasNewComments to this post's .thing
+					$link.closest('.thing').addClass('res-hasNewComments');
```

Comment:
Maybe; the [selector](https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/2958/files#diff-45da8d92031230da79b8bcf0afcfe1aeL270) above guarantees that we'll be within a `.thing`, at least.